Method on NDCube for showing physical axes for each array axis (#281)
sunpy/ndcube: changelog/281.feature.rst

> New method on NDCube (array)axis_physical_types) to show which physical types are associated with each array axis.

Author: ```suggestion
New method on NDCube (array_axis_physical_types) to show which physical types are associated with each array axis.
```

> New method on NDCube (array_axis_physical_types) to show which physical types are associated with each array axis.

Reviewer: ```suggestion
Added a new method `ndcube.NDCube.array_axis_physical_types` to show which physical types are associated with each array axis.
```
